en/micro-integrator/docs/references/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+# AmazonSQS Inbound Endpoint Example
+
+The AmazonSQS inbound endpoint allows you to consume messages via WSO2 EI. The inbound endpoint acts as a message consumer. It creates a connection to an Amazon SQS account, consumes messages from the Amazon SQS queue, and injects the messages into the WSO2 EI sequence.
+
+## What you'll build
+
+This scenario demonstrates how the AmazonSQS inbound endpoint works as an message consumer. In this scenario, you should have a connectivity with Amazon AWS account. Please follow the steps mentioned in the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) document in order to create a Amazon account and obtain access key id and secret access key.
+                                                                                                                                                                      
+The AmazonSQS inbound endpoint is listening to the `Simple Queue Service` which is created under `Application Integration` section in the AWS Services. First you need to create a **Queue** inside the **Simple Queue Service** and send a message to the created Queue. The WSO2 EI AmazonSQS Inbound Endpoint will receive the message and notify. Then just log the SMS message content. In your own scenarios, you can inject that message into the mediation flow for getting the required output.
```

Comment:
notify --> log message content to the console?

Reply:
If you are extending this sample scenario, you can perform any kind of mediation using the [mediators](https://ei.docs.wso2.com/en/latest/micro-integrator/references/mediators/about-mediators/) available with WSO2 EI.

Reply:
`Then just log the SMS message content.` This is copied over. Pls remove.

Reply:
Updated.